advocacy_docs/edb-postgres-ai/analytics/concepts.mdx

```diff
@@ -7,45 +7,45 @@ description: Learn about the ideas and terminology behind EDB Postgres Lakehouse
 EDB Postgres Lakehouse is the solution for running Rapid Analytics against
 operational data on the EDB Postgres® AI platform.
 
-## Major Concepts
+## Major concepts
 
-* **Lakehouse Nodes** query **Lakehouse Tables** in **Managed Storage Locations**.
-* **Lakehouse Sync** can create **Lakehouse Tables** from **Transactional Tables** in a source database.
+* **Lakehouse nodes** query **Lakehouse tables** in **managed storage locations**.
+* **Lakehouse sync** can create **Lakehouse tables** from **Transactional tables** in a source database.
```

Comment:
```suggestion
* **Lakehouse Sync** can create **Lakehouse tables** from **Transactional tables** in a source database.
```

Lakehouse Sync is a specific component of Lakehouse and will be getting its own sections.